### ISSUE TITLE
Foobar v1.6 update causing immediate crash

### ISSUE THREAD

**Issue by Madmax4434** [kbuffington/Georgia#48]
Seems to start up fine in safe mode but crashes immediately when updating to 1.6 and starting with Georgia applied.

**Comment by mist475** [comment]
this is because of jscript panel being incompatible, if you update jscriptpanel you can start the program, you can't play anything though

**Comment by Madmax4434** [comment]
Checked the foo jscript panel version and I'm on the latest. Pretty sure it isn't that since it was working up until I updated

**Comment by mist475** [comment]
hmm, odd. for me it launches fine, then the jscript panel turns red, you sure you're on 2.4.1.1?

**Comment by Madmax4434** [comment]
Seems I was on 2.3.6.1 from kbuffington, updated to 2.4.1.1 and facing the same issue as you now. Rolled back to foobar 1.5.6 and it's working fine though.

**Comment by kbuffington** [comment]
Ugh, of course this would happen before I'm ready to release the latest version. I'll take a look. Not sure what changed since the last beta I was working with.

**Comment by kbuffington** [comment]
I cannot get this to crash using either version 2.4.0 or 2.4.1.1. There might have been something fixed since the 1.1.9 release?
What does the console say?

**Comment by Madmax4434** [comment]
With jscript panel 2.3.6.1, it's stable on 1.5.6 but crashes on 1.6 foobar. With 2.4.1.1 it should turn red on both if you try to perform any activity. Try editing a song property for example.

**Comment by bobstrain** [comment]
This is the console output on error:

`JScript Panel v2.4.1.1 (Georgia by Mordred)
Errore di run-time di JavaScript:
Azione non valida per l'oggetto
File: C:\Users\...\AppData\Roaming\foobar2000\georgia\js\CaTRoX_QWR\Panel_Library.js
Line: 2697, Col: 3
`

For what I understand it's related to window.CreateTooltip, from marc2k3 notes on 2.4.1:

> Update docs to make it clear that only a single call to window.CreateTooltip is permitted per panel instance. Errors will now be thrown on any subsequent calls. Additionally, the ITooltip interface no longer has a Dispose method.

Hope this help

**Comment by BenGroeneveld** [comment]
I am experiencing this problem as well (foobar2000 1.6, Georgia 1.1.9 and JScript Panel v2.4.1.1).

The theme will load, but it will give an JavaScript error within a few seconds, seemingly regardless of any actions.

I received this error message:
```
JScript Panel v2.4.1.1 (Georgia by Mordred)
JavaScript runtime error:
Object doesn't support this action
File: C:\Users\[USERNAME]\AppData\Roaming\foobar2000\georgia\js\CaTRoX_QWR\Panel_Library.js
Line: 2697, Col: 3
```

Besides that, I also noticed there is no option to select the "Chakra" (or any other) script engine in JScript Panel v2.4.1.1, not sure if this is helpful, but I thought I'd share.

**Comment by kbuffington** [comment]
Should be fixed in 1.1.10
